qemu/tests/boot_N_1_virtserialports.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+from virttest import error_context
+from virttest import env_process
+from virttest import utils_test
+from qemu.tests.virtio_serial_file_transfer import transfer_data
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Boot guest with multiple virtio-serial-devices
+
+    1. Boot a guest with 26 virtio-serial-pcis
+    2. Every bus is used for one virtserialport
+    3. Transfer data via evey port
+
+    :param test: kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    num_bus = params.get_numeric("numberic_bus")
+    for i in range(2, num_bus + 1):
+        serial_name = 'vs%d' % i
+        params['serials'] = '%s %s' % (params.get('serials', ''), serial_name)
+        params['serial_type_%s' % serial_name] = "virtserialport"
+        params['serial_reuse_bus_%s' % serial_name] = "no"
+    params['start_vm'] = "yes"
+    env_process.preprocess(test, params, env)
```

Comment:
How do you think of use different branch according to the value of `numberic_bus`. The code above only work when `numberic_bus` is defined.   Others reuse the existing `boot_1_N_virtserialports.py`.

Reply:
I think we can use the value of `start_vm` as a flag.

Reply:
That is ok for me too, can distinguish this case with others is ok.

Reply:
ok, it seems better that we all use `boot_N_M_virtserialports.py` file since N and M are more flexible.

Reply:
Hi nana, the code below is the same with qemu/tests/boot_N_M_virtserialports.py , how about making them in one py file. You can add if condition for the above code for n:1 scenario.
Btw, name N_M can include n:1, 1:n, n:m scenarios.

@yanan-fu  How do you think?

Reply:
@xiagao We all have same idea as the comments above. Nana will update it after https://github.com/autotest/tp-qemu/pull/2198 be merged. thanks!